frontend/pages/404.tsx

```diff
@@ -0,0 +1,54 @@
+import { Center, Text, Divider, VStack } from '@chakra-ui/react';
+import { WithProvidersProps } from '@components/common';
+import {
+   DefaultLayout,
+   getLayoutServerSideProps,
+   WithLayoutProps,
+} from '@layouts/default';
+import { GetStaticProps } from 'next';
+
+type Custom404Props = WithProvidersProps<WithLayoutProps<{}>>;
+
+export const Custom404: NextPageWithLayout<Custom404Props> = () => {
+   return (
+      <Center flexGrow={1}>
```

Comment:
Now that I'm looking at this again, I think we should add `flexGrow={1}` to `frontend/layouts/default/DefaultLayout/index.tsx`. We should wrap the children in a `Box` with that property. So it would look like

```tsx
<Box flexGrow={1}>
   {children}
</Box>
```

That way, any page that has too little content to fill the viewport will grow to fill the viewport.

Reply:
It looks like adding that change borked some of the components on `/Tools` and `/Parts`. I'm going to un_dev_block :thought_balloon: on this unless someone else thinks we should add this